unified-codes/release.sh

```diff
@@ -0,0 +1,30 @@
+### unified codes release steps                     ###
+### before using, add your public key to the server ###
+### to allow scp & ssh                              ###
+
+################ web ################
+ cp .dist.env .env
+ mv .local.env .local.env.backup
+
+ nx build web --prod --optimization
+ rm .env
+
+ scp ./dist/apps/web/* root@codes.msupply.foundation:/var/www/unified-codes/web/
```

Comment:
Could set up some variables at the top-level scope? E.g.

```
scp $PATH_WEB_APP $USER_ROOT@$BASE_URL:$PATH_WEB_WWW
```

Like that, but something better than that. 

![seeya](https://user-images.githubusercontent.com/43223637/99312813-77e28600-28c3-11eb-8638-070ed8d37918.gif)